packages/sources/src/components/definition-list/definition-list.models.ts

```diff
@@ -6,5 +8,13 @@ export interface DefinitionList<TemplateFnReturnType = never> {
 
 export interface Definition<TemplateFnReturnType = never> {
   term: string;
-  descriptions: (string | TemplateFnReturnType)[];
+  descriptions: (DefinitionDescriptionContent<TemplateFnReturnType> | DefinitionDescriptionItems)[];
+}
+
+export interface DefinitionDescriptionContent<TemplateFnReturnType> {
+  content: string | TemplateFnReturnType;
+}
+
+export interface DefinitionDescriptionItems {
+  columnsList: Lists;
```

Comment:
De naam van het model `Lists` klopt niet. Ik begrijp wel waar de verwarring vandaan komt, de interface die we 'n paar maanden geleden hadden geschreven ging nog uit van een component `ColumnList`. Maar dit is tenminste enkelvoud, en nu maak jij meervoud.

Als de naam van dit component is verbeterd, kan dat teruggevoerd worden naar de property name.

Moet dit niet gewoon "List" zijn?